destination/config.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+//go:generate paramgen -output=paramgen.go Config
+
+type Config struct {
+	// amazon access key id
+	AWSAccessKeyID string `json:"aws.accessKeyId" validate:"required"`
+	// amazon secret access key
+	AWSSecretAccessKey string `json:"aws.secretAccessKey" validate:"required"`
+	// amazon sqs region
+	AWSRegion string `json:"aws.region" validate:"required"`
+	// amazon sqs queue name
+	AWSQueue string `json:"aws.queue" validate:"required"`
+	// amazon sqs message delay time
+	AWSSQSMessageDelay int32 `json:"aws.delayTime"`
```

Comment:
Sine this option is not required, what's the default value?

Reply:
I'll add the default to be 0! (for the one above too)